run_iwyu_tests.py

```diff
@@ -83,26 +83,29 @@ def setUp(self):
       'prefix_header_includes_remove.cc': ['--prefix_header_includes=remove'],
       'prefix_header_operator_new.cc': ['--prefix_header_includes=remove'],
     }
-    prefix_headers = [self.Include('prefix_header_includes-d1.h'),
+    prefix_headers = ['-I .', 
+                      self.Include('prefix_header_includes-d1.h'),
                       self.Include('prefix_header_includes-d2.h'),
                       self.Include('prefix_header_includes-d3.h'),
                       self.Include('prefix_header_includes-d4.h')]
     clang_flags_map = {
-      'alias_template.cc': ['-std=c++11'],
+      'alias_template.cc': ['-I .', '-std=c++11'],
       'auto_type_within_template.cc': ['-std=c++11'],
       # MSVC targets need to explicitly enable exceptions, so we do it for all.
-      'catch.cc': ['-fcxx-exceptions', '-fexceptions'],
-      'clmode.cc': ['--driver-mode=cl', '/GF', '/Os', '/W2'],
-      'conversion_ctor.cc': ['-std=c++11'],
+      'catch.cc': ['-I .', '-fcxx-exceptions', '-fexceptions'],
+      'clmode.cc': ['-I .', '--driver-mode=cl', '/GF', '/Os', '/W2'],
+      'conversion_ctor.cc': ['-I .', '-std=c++11'],
       'deleted_implicit.cc' : ['-std=c++11'],
       'lambda_fwd_decl.cc': ['-std=c++11'],
-      'lateparsed_template.cc': ['-fdelayed-template-parsing'],
+      'lateparsed_template.cc': ['-I .', '-fdelayed-template-parsing'],
       'ms_inline_asm.cc': ['-fms-extensions'],
       'prefix_header_attribution.cc': [
-          self.Include('prefix_header_attribution-d1.h')],
+          '-I .', self.Include('prefix_header_attribution-d1.h')],
       'prefix_header_includes_add.cc': prefix_headers,
       'prefix_header_includes_keep.cc': prefix_headers,
       'prefix_header_includes_remove.cc': prefix_headers,
+      'new_header_path_local.cc': [''], # no include paths
```

Comment:
Ok I stated it a bit wrong. `-I .` was not the default but a forced setting. So I changed it to be the default, to make as few changes as possible. A more reasonable default value would be an empty list.
But sure, we could also add another param for include paths to make it cleaner (only `.` instead of `-I .`). I still think the default should be empty...